### DISCUSSION TITLE
Strange issue on Updates

### DISCUSSION THREAD

**Discussion by cmh714** [rern/rAudio#1036] [Fixed Issues]
I have had this problem for the last several weeks and have thus far not been able to fix it. Basically, everytime I reboot, the Update shows that I need to download it. However, I already have the update.

At first, my r1 file showed the wrong date, so I adjusted it. Next reboot, it showed that I needed an update, and the r1 file had an older date. Its super annoying, so if anyone has an idea, that would be appreciated!!

And like now, it says I need to update, the r1 file is showing a date older than 20220917, but I already have this update. Do the update, the r1 file is set correctly to 20220917......I can guarantee that the next reboot will have a date prior to that....

Like I said, not the largest problem but still a PITA! TIA!

**Comment by cmh714** [comment]
Tried running the command by hand, still stuck at 20220904......its insane!

rAudio:/tmp  #  ./install.sh "r1 Update 20220917"
jq: error: Could not open file Update: No such file or directory
jq: error: Could not open file 20220917.title: No such file or directory

================
 .   {
  "title": "rAudio 1",
  "version": "20220917",
  "revision": [
    "Fix - Library - Web Radio multi-level subdirectories",
    "Fix - Web Radio - Broken Playback when add new station before updating Library database",
    "Fix - (RPi Zero and 1) Networks - Broken page if <code>bluez</code> not upgraded"
  ],
  "description": "Updates for rAudio 1",
  "buttonlabel": "Update",
  "nouninstall": 1,
  "thumbnail": "/assets/img/coverart.svg",
  "sourcecode": "https://github.com/rern/rAudio-1",
  "installurl": "https://github.com/rern/rAudio-1/raw/main/install.sh",
  "verify": {
    "command": "[[ $( cat /srv/http/data/addons/r1 ) < 20220805 ]] && echo 1",
    "notverified": "Please download the latest image file to be able to update."
  }
} ...
=================
 .  Get files ...
jq: error: Could not open file Update: No such file or directory
jq: error: Could not open file 20220917.installurl: No such file or directory
curl: option -: is unknown
curl: try 'curl --help' for more information
 !  Get files failed.

**Comment by cmh714** [comment]
Ah, just disregard.....I chose to fresh flash and restore. :)

**Reply by SejaMarginal** [reply]
I have the same problem and I don't want to do a fresh install because I have a lot to configure.
So I would love to know how to update from command line. 
The Update button on UI isn't working.

**Reply by cmh714** [reply]
As I wrote, I had to reimage to fix the problem. Just make a backup and restore it on the new image

**Reply by SejaMarginal** [reply]
My problem is that I have a display, volume configuration and bluetooth dongle to recompile/configure within a new installation.

**Reply by cmh714** [reply]
Ouch! Yeah, thats a lot to fix.....I tried everything I could think of before re-imaging but nothing worked. GL!!
